Tests/Helper/LocaleHelperTest.php

```diff
@@ -35,6 +35,7 @@ public function testLanguage()
         $helper = $this->getHelper();
         $this->assertEquals('français', $helper->language('fr'));
         $this->assertEquals('français', $helper->language('fr_FR'));
+        $this->assertEquals('anglais américain', $helper->language('en_US'));
```

Comment:
Isn't line 37 already testing the piece of code you changed ? I don' t think it is possible to get a test that fails without your change, actually.